argo-cd-apps/base/all-clusters/infra-deployments/backup/backup.yaml

```diff
@@ -9,6 +9,9 @@ spec:
           - nameNormalized
         generators:
           - clusters:
+              selector:
+                matchLabels:
+                  appstudio.redhat.com/backup: 'true'
```

Comment:
It does not work, the matchLabel expression is overwritten by the all-cluster selector. I found a way to get it to work with the explanation, see https://github.com/redhat-appstudio/infra-deployments/pull/3675